package.json

```diff
@@ -38,7 +38,7 @@
   "devDependencies": {
     "babel-eslint": "^10.0.3",
     "electron": "^4.2.11",
-    "electron-builder": "^21.2.0",
+    "electron-builder": "^20.44.4",
```

Comment:
why the downgrade?

Reply:
better error log when trying to build locally, i can rollback this now